src/korotkov_nfa.cpp

```diff
@@ -120,7 +124,13 @@ void nextStep(std::stack<keyState *>& stack, keyState* input)
     default:
             stack.push(input);
             break;
-    case Split:
+    case SplitA:
+            e1 = new keyState{input->qGramFrag_, itptr->out1_, nullptr};
+            e2 = new keyState{input->qGramFrag_, itptr->out2_, nullptr};
+            stack.push(e2);
+            stack.push(e1);
```

Comment:
if you change the SplitB the same way, there is no need for `keyState *e1, *e2`.
```suggestion
            stack.push(new keyState{input->qGramFrag_, itptr->out2_, nullptr});
            stack.push(new keyState{input->qGramFrag_, itptr->out1_, nullptr});
```

Reply:
Ah whoops, I actually plan to totally get rid of this module! it's Vincent's...